packages/sshnoports/lib/sshnp/sshnp_params.dart

```diff
@@ -230,17 +230,17 @@ class SSHNPPartialParams {
     this.clientAtSign,
     this.sshnpdAtSign,
     this.host,
-    this.device,
-    this.port,
-    this.localPort,
+    this.device = SSHNP.defaultDevice,
```

Comment:
SSHNPPartialParams should not use default or fallback values, it breaks the functionality of the merge constructors for both SSHNPPartialParams and SSHNPParams